### ISSUE TITLE
ChipGroup  - prop for heading level

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#2167]
A new prop `headingLevel` should be added to the ChipGroup component to allow the user to specify heading levels. Supported values should be `h1` – `h6`.

This would affect the heading element that is used for `.pf-c-chip-group__label` in the example below:
```
<ul class="pf-c-chip-group pf-m-toolbar">
  <li>
    <h4 class="pf-c-chip-group__label">Category 1</h4>
  ...
```

Similar to my comment in #1881 for Accordion, I think this prop belongs on the parent `<ChipGroup>` component so that the same level is used for all categories. I don't see a use case where these should be different heading levels.

And assuming we land with defaulting to `<h1>` for #1881, then we should be consistent and do that here as well.
